core-plugins/src/main/java/io/cdap/plugin/alert/TMSAlertPublisher.java

```diff
@@ -136,13 +144,15 @@ private Conf() {
       maxAlertsPerSecond = 100;
     }
 
-    private void validate() {
+    private void validate(FailureCollector collector) {
       if (autoCreateTopic && namespace != null) {
-        throw new IllegalArgumentException("Cannot auto-create topic when namespace is set.");
+        collector.addFailure("Cannot auto-create topic when namespace is set.",
+                             "Namespace must not be set to auto-create topic.").withConfigProperty("namespace")
+                .withConfigProperty("autoCreateTopic");
```

Comment:
method calls should be indented 2 spaces.

```
collector.addFailure("...",
                     "...")
  .withConfigProperty("namespace")
  .withConfigProperty("autoCreateTopic")
```

Reply:
Done